mock/environment/__init__.py

```diff
@@ -16,34 +17,49 @@
 class Env:
    
     def __init__(self):
+        """
+        Stores weather data into a dictionary 
+        that can be indexed by site and date.
+        """
         self.site_data = {}
         self.site_data_by_night = {}
-        
+
         for site in Site:
             site_lc = site.name.lower()
             input_filename = Env.data_file_path(f'{site_lc}_weather_data.pickle.bz2')
```

Comment:
Also, to make things simpler, I moved `data_file_path` into `__init__`. (In Python, you can have functions inside functions.) We'll never use it anywhere else, so we may as well put it here.